molo/profiles/templates/profiles/viewprofile.html

```diff
@@ -3,22 +3,26 @@
 {% get_settings %}
 
 {% block content %}
-
-<div class="profile-style">
-
-  <h3>{% trans "Hello " %}{{ user.username }}</h3>
-  <label class="label-wide">{% trans "Display Name: " %}{{ user.profile.alias | default:'Anonymous' }}</label>
-  <label class="label-wide">{% trans "Date of Birth: " %}{{ user.profile.date_of_birth | date:"M d, Y" }}</label>
+<h1 class="heading heading--profiles">{% trans "Hello " %}{{ user.username }}</h1>
+<div class="profiles">
+  <p>{% trans "Display Name: " %}{{ user.profile.alias | default:"Anonymous" }}</p>
+  <p>{% trans "Date of Birth: " %}{{ user.profile.date_of_birth | date:"M d, Y" }}</p>  
   {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
-    <label class="label-wide">{% trans "Mobile Number: " %}{{ user.profile.mobile_number | default:'' }}</label>
+  <fieldset>
+      <label for="id_profile.mobile_number">{% trans "Mobile Number: " %}</label>
+      {{ user.profile.mobile_number | default:"" }}
+  </fieldset>
   {% endif %}
+
   {% if settings.profiles.UserProfilesSettings.show_email_field %}
-    <label class="label-wide">{% trans "Email Address: " %}{{ user.email | default:'' }}</label>
+  <fieldset>
```

Comment:
@Mitso @peterp here as well, I think they should be same as `Display Name` and `Date of Birth`